src/tests/system/tests/test_access_control_ldap_filter.py

```diff
@@ -0,0 +1,251 @@
+"""
+SSSD Authentication Test Cases
+
+:requirement: access control access_filter
+
+LDAP Filters are not intenteded to work with IPA provider
```

Comment:
It's a lot of verbiage, how about,

```The IPA provider does not support LDAP filters; IPA tests are skipped.```

If people are trying to find an alternative solution for access control in IPA, it will be a long time before they get here.